planner-api/src/main/java/pl/edu/agh/kuce/planner/event/persistence/OneTimeEventRepository.java

```diff
@@ -7,6 +7,6 @@
 
 public interface OneTimeEventRepository extends JpaRepository<OneTimeEvent, Integer> {
 
-    @Query("SELECT o FROM OneTimeEvent o WHERE o.user.nick = ?1")
-    Collection<OneTimeEvent> findByUsersNick(String userNick);
+    @Query("SELECT o FROM OneTimeEvent o WHERE o.user.id = ?1")
```

Comment:
I'd suggest replacing this method with findByUser(User user). Query annotation can be omitted in that case. I think it'd be simpler and cleaner to pass User object than its ID.